packages/client/hmi-client/src/components/model/petrinet/tera-line-graphic.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<div ref="lineGraphic">
+		<svg :height="parentHeight">
+			<line x1="0" :y1="parentHeight / 2" :x2="width" :y2="parentHeight / 2" stroke="black" />
+			<rect
+				v-if="distribution.type === DistributionType.Uniform"
+				:x="rectX"
+				:y="parentHeight / 2 - 2"
+				:width="rectWidth"
+				rx="2"
+				height="4"
+				stroke="gray"
+				fill="gray"
+			/>
+			<circle v-else :cx="circlePoint" :cy="parentHeight / 2" r="4" stroke="black" fill="black" />
+		</svg>
+	</div>
+</template>
+
+<script setup lang="ts">
+import * as d3 from 'd3';
+import { onMounted, ref } from 'vue';
+import { DistributionType } from '@/services/distribution';
+
+const props = defineProps<{
+	maxValue: any;
+	minValue: any;
+	distribution: any;
+}>();
```

Comment:
Is it possible to tighten up the props minValue, maxValue, distribution? 

Below you are treating them as if they have types so can we just do the typings here?

Reply:
will do